src/interactive-window/outputs/tracebackFormatter.ts

```diff
@@ -30,9 +30,10 @@ export class InteractiveWindowTracebackFormatter implements ITracebackFormatter
         if (!storage) {
             return traceback;
         }
+        const useIPython8Format = traceback.some((traceFrame) => /^[Input|File].*?\n.*/.test(traceFrame));
         return traceback.map((traceFrame) => {
             // Check IPython8. We handle that one special
-            if (/^[Input|File].*?\n.*/.test(traceFrame)) {
+            if (useIPython8Format) {
```

Comment:
This was incorrect, we need to test the regex on the entire output, not just one line.